less/adjacent-nav.less

```diff
@@ -1,36 +1,49 @@
 
 nav.adjacent {
+  // Display & Visibility
```

Comment:
Respectfully requesting that these inline comments be removed. They introduce a lot of noise into the stylesheets and I will 100% forget to include them with new selectors.

It's a fine idea to organize properties, but I'm pretty sure we could do this automatically with tooling — some sort of PostCSS hook or something.

Reply:
That's honestly totally valid. I liked having them organized like this with breaks between each property, made things especially easy when combing those several documents into `base.less` but great point in future maintenance this would likely be a pain.

I'll go ahead and remove them, and instead we can implement this breakup of properties as a GitHub Action whenever it's moved over to it's own repo.

Reply:
I've gone ahead and removed all the comments. You'll notice I left the empty newlines in places where the selector has more than 5 attributes. I thought it helped breakup the flow between them for easier reading, but let me know if you still find it noisy.

But otherwise I'll start looking into tooling to organize this, and ideally keep the new lines in as well if we like them